src/Fetch.fs

```diff
@@ -11,6 +11,28 @@ open Browser.Types
 
 [<AutoOpen>]
 module Types =
+    [<StringEnum; RequireQualifiedAccess>]
+    type RequestDestination =
+        | [<CompiledName("")>] Empty
+        | Audio
+        | [<CompiledName("audioworklet")>] AudioWorklet
```

Comment:
Probably we need to have a `LowerAll` Case rule so we don't need to use CompiledName in this situations.